conda_forge_tick/profiler.py

```diff
@@ -0,0 +1,34 @@
+import cProfile
+import os
+from functools import wraps
+from datetime import datetime
+
+
+# profiling decorator
+def profiling():
+    def _profiling(f):
+        @wraps(f)
+        def __profiling(*rgs, **kwargs):
+            with cProfile.Profile() as prof:
+                prof.enable()
```

Comment:
Do you still need the enable and disable?

Reply:
not anymore, but I will read again the doc about `cProfile.Profile()` to check if there is any problem reggarding that (like the need of `prof.disable`).

Reply:
Yup, I've checked [here](https://bugs.python.org/issue29235) an it seems to be unnecessary (redundant and wrong) the usage of the enable and disable in this case. Thanks for the remind CJ.

Reply:
tested it locally, seams to be working fine. I will update the PR

Reply:
Done